genny/config/templates/postgres.yml.tmpl

```diff
@@ -1,6 +1,6 @@
 ---
 development:
-  dialect: postgres
+  dialect: pgx
```

Comment:
`postgres` and `pgx` are declared as synonyms in the postgres driver. I don't think you need this change. :)